pkg/gcs/gcs.go

```diff
@@ -31,21 +31,21 @@ import (
 	"google.golang.org/api/option"
 )
 
-type Client struct {
+type storageClient struct {
 	*storage.Client
 }
 
 // NewClient creates new GCS client with given service account
-func NewClient(ctx context.Context, serviceAccount string) (*Client, error) {
```

Comment:
Could you confirm the returned value is properly changed to be a value and not a reference?

Reply:
`Client` is now an interface, and `*storageClient` has implemented the interface, so here it's returning a `*storageClient` as the `Client` type.